<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-3HDS7VK" failures="1" tests="5" name="Test" time="0.222" errors="0" timestamp="14 Mar 2021 18:02:39 GMT">
  <testcase classname="testng.tasks.TestNgTaskForClarification" name="chockychocky" time="0.037"/>
  <testcase classname="testng.tasks.TestNgTaskForClarification" name="kinderJoy" time="0.006"/>
  <testcase classname="testng.tasks.TestNgTaskForClarification" name="lollypop" time="0.009"/>
  <testcase classname="testng.tasks.TestNgTaskForClarification" name="geams" time="0.007"/>
  <testcase classname="testng.tasks.TestNgTaskForClarification" name="cadberry" time="0.017">
    <failure type="java.lang.AssertionError" message="execution getting fails expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: execution getting fails expected [true] but found [false]
at testng.tasks.TestNgTaskForClarification.cadberry(TestNgTaskForClarification.java:20)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- cadberry -->
</testsuite> <!-- Test -->
